release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -454,6 +454,11 @@ With this update, you can now create multi-network policies for IPv6 networks. F
 [id="ocp-4-15-ingress-dashboards"]
 ==== Ingress Operator metrics dashboard available
 With this release, Ingress networking metrics are now viewable from within the {product-title} web console. See xref:../networking/networking-dashboards.adoc#ingress-dashboards[Ingress Operator dashboard] for more information.
+[id="ocp-4-15-CoreDNS-filtration-externalname-service-queries"]
```

Comment:
I see that this renders fine, but I still think it's a best practice to leave a blank line in between the previous sentence and this ID line.

Reply:
Updated As Suggested.